### PR TITLE
[change] `tvdb_lookup` - Always have the main poster as the first entry of `tvdb_posters`

### PR DESCRIPTION
**Pull request by BrutuZ**
### Motivation for changes:
TheTVDB returns posters in a random order and Flexget caps the list to 5 entries. I like to use the main poster (curated to be used by the site) on my notifications and given the aforementioned actions, it's not always on the list, and when it is there is no way to determine which entry t is.
### Detailed changes:
- ~~[unrelated] Changed URL for the posters CDN to the new one post TheTVDB's v3 migration. There are currently 301 redirects in place at the old URL but they did recommend changing it wherever possible.~~
- Request the main poster path from the `/series/{id}` endpoint and start the list with it
- Append up to 5 posters to the list (not 4 just in case the main one is not defined)
- Trim the list down to 5 entries

### Addressed issues:
- Fixes *my OCD*.

### CODE REVIEW

File: flexget/components/thetvdb/api_tvdb.py [hunk 1]
```diff
@@ -257,10 +257,12 @@ def get_posters(self):
         if not self._posters:
             log.debug('Getting top 5 posters for series %s', self.name)
             try:
+                poster_main = TVDBRequest().get('series/%s' % self.id).get('poster')
```
Comment (liiight) [inline flexget/components/thetvdb/api_tvdb.py]: please use fstring

Comment (BrutuZ) [reply flexget/components/thetvdb/api_tvdb.py]: Sure thing. Only this new line or should I go ahead and convert all other `%s` replacements on the file for f-strings too?

Comment (liiight) [reply flexget/components/thetvdb/api_tvdb.py]: All please 😁

Comment (BrutuZ) [reply flexget/components/thetvdb/api_tvdb.py]: On it 👍 .
Just one more question (I hope): Leave log entries alone to let it handle the parameters internally or convert them to f-strings too?

**Comment by liiight** [comment]
Yes, leave logger to do it's own interpolation

**Comment by liiight** [comment]
Thanks for this!

**Pull request merged**